docs/reference/commandline/cli.md

```diff
@@ -46,6 +46,8 @@ by the `docker` command line:
   unsuitable for Docker.
 * `DOCKER_RAMDISK` If set this will disable 'pivot_root'.
 * `DOCKER_TLS_VERIFY` When set Docker uses TLS and verifies the remote.
+* `DOCKER_TRUST` When set Docker uses notary to sign and verify images.
+  Equates to `--untrusted=false` for build, create, pull, push, run.
```

Comment:
Customizing the `DOCKER_TRUST_SERVER` is something we purposely left out of the documentation. This was only added for testing and is not currently a work flow we want to document and market. Do you think despite this it should be documented here? Same could be said of the `*_PASSPHRASE` environment variables.